cmd/antrea-controller/controller.go

```diff
@@ -114,6 +115,10 @@ func run(o *Options) error {
 		traceflowController = traceflow.NewTraceflowController(crdClient, podInformer, traceflowInformer)
 	}
 
+	// statsAggregator takes NetworkPolicy metrics summaries pushed by antrea-agents, aggregates them, and serves the
+	// NetworkPolicy metrics API with the aggregated data.
+	statsAggregator := stats.NewAggregator(networkPolicyInformer, cnpInformer)
```

Comment:
No strong opinion. I am perfectly fine with using the same components for all metrics here, and it's probably better to avoid multiplying APIs. Maybe a rephrase of the comment would be nice to avoid making it NP-specific, and we can mention at the end of the comment that for now we only expose NP stats.

Reply:
Rephrased the comment as you suggested, thanks.